lib/browser/api/menu-item.js

```diff
@@ -11,7 +11,7 @@ const MenuItem = function (options) {
   for (let key in options) {
     if (!(key in this)) this[key] = options[key]
   }
-
+  this.submenu = this.submenu || roles.getDefaultSubmenu(this.role)
```

Comment:
Can you add a spec for these new cases in the https://github.com/electron/electron/blob/master/spec/api-menu-spec.js#L424 block?